.github/scripts/test-collector-ctl.sh

```diff
@@ -86,10 +86,18 @@ test_collector_ctl_with_samecfg_restart() {
     echo "${FUNCNAME[0]} ... OK"
 }
 
+test_collector_ctl_with_featureflag() {
+    #staart coll default conf by starting without -c
+    $ADOT_CTL -a start -f "-adot.exporter.datadogexporter.deprecation"
+
+    echo "${FUNCNAME[0]} ... OK"
+}
+
 setup
 
 ## Tests
 test_collector_ctl_does_not_overwrite_env
 test_collector_ctl_with_sed_special_chars
 test_collector_ctl_with_samecfg
 test_collector_ctl_with_samecfg_restart
+test_collector_ctl_with_featureflag
```

Comment:
if we also run these tests with windows powershell script, that's great. if not, we should probably do so.